shared/macros.jinja

```diff
@@ -1827,3 +1827,21 @@ Fill in <tt>GRUBENV_FILE_LOCATION</tt> based on information above.
     For each kernel installed, a line with value "{{{ value }}}" should be returned.
     {{% endif %}}
 {{%- endmacro %}}
+
+
+{{#
+Macro describing fix for sysctl rules.
+:param sysctl: The kernel parameter to be changed.
+:type sysctl: str
+:param value: The value of the kernel parameter.
+:type value: str
```

Comment:
Please add a blank link before the closing tag of the docstring. See the [style guide](https://complianceascode.readthedocs.io/en/latest/manual/developer/04_style_guide.html#jinja).